docs/developer-interface.rst

```diff
@@ -13,8 +13,8 @@ Client
 .. autoclass:: tamr_unify_client.Client
   :members:
 
-Attribute
----------
+Attributes
```

Comment:
Duplicate labels (e.g. this section header and the very next one are both called `Attribute`) cause Sphinx warnings, which we turn into errors.

Reply:
🤦‍♂